### PR TITLE
Milestone2

### PR DESCRIPTION
**Pull request by Ashah15**
initialize the Redux actions, reducers, and store. And define empty React components, separate presentation, and container components.

### CODE REVIEW

File: src/reducers/initialState.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+const initialState = [
+  { id: 0, title: 'Friday 13', category: 'Horror' },
+  { id: 1, title: 'Wild-life Dangers', category: 'Learning' },
+  { id: 2, title: 'Toonami', category: 'Kids' },
```
Comment (bellom) [inline src/reducers/initialState.js]: ### [Optional]
Kindly be aware that you can use uuidv4 to automatically generate ids. read more 👉 [npm uuid](https://www.npmjs.com/package/uuidv4)

Example:
```
import uuid from 'uuidv4'
id: uuid(): {title: 'Friday 13', category: 'Horror'}
```

File: src/reducers/initialState.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+const initialState = [
```
Comment (bellom) [inline src/reducers/initialState.js]: According to the instructions on pathwright, I think you need to move this to `src/index.js`

image

[Pathwright](https://microverse.pathwright.com/library/react-redux-v11/184438/path/step/106376028/)

**Review by bellom — CHANGES_REQUESTED** [review]
## Hi @Ashah15  :clap: 

### Awesome job so far!! :+1: 

#### You have done wonderfully well, you passed `almost` all the checklist for this project :+1: just a few issues left to work on............ :slightly_smiling_face: 

# Status: Changes Required! ♻️ 

**1. Kindly check out my comments below;**

**2. Kindly update your `README` file**
**Let's make your repo look more professional by updating your `README` file**.
Checkout out the below link for help, feel free to follow this [Microverse template](https://github.com/microverseinc/readme-template) from pathwright

### ♻️ Kindly re-submit for review after making all those changes :slightly_smiling_face: Thumps -up await you.....

**Best,**

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

**Review by uche-inyama — CHANGES_REQUESTED** [review]
## Changes Required

### Hi Asha Abdulah Hassan :slightly_smiling_face: 

Kindly update your readme, as required from the last review.

### Have a great day :100: 

#### ``` Uchechukwu Henry Inyama ```

**Review by taiwo2 — APPROVED** [review]
# STATUS: APPROVED
@Ashah15  Weldone for implementing the suggestion from previous Code Reviewer. 
Your project is approved kindly make sure your deployment works fine in your next milestone task. Thanks

**Pull request merged**